bitnami/external-dns/templates/deployment.yaml

```
@@ -635,6 +635,7 @@ spec:
                   key: infoblox_wapi_password
             {{- end }}
             {{- end }}
+            {{- if eq .Values.provider "rfc2136" }}
             {{- if .Values.rfc2136.tsigSecret | or (and .Values.rfc2136.kerberosUsername .Values.rfc2136.kerberosPassword) | or .Values.rfc2136.secretName }}
```

I'd include the new condition inside of the existing one 

```suggestion
            {{- if and ( eq .Values.provider "rfc2136" ) ( .Values.rfc2136.tsigSecret | or (and .Values.rfc2136.kerberosUsername .Values.rfc2136.kerberosPassword) | or .Values.rfc2136.secretName ) }}
```

---

You are completely right. In that case, to follow the same syntax, could you please revert the change so I accept it and merge the changes? Sorry for the noise